cernopendata/modules/fixtures/data/records/opera-ecc-datasets.json

```diff
@@ -247,7 +247,7 @@
       "attribution": "CC0"
     },
     "methodology": {
-      "description": "FIXME"
+      "description": "<p>Events stored in this dataset were collected between 2008 and 2012. Events are classified as “1mu” if one track is tagged as a muon after the analysis of the electronic detectors data, “0mu” otherwise. In this sample “0mu” events and “1mu” events with a muon momentum lower than 15 GeV/c were selected since they are richer in terms of their possible tau neutrino content.</p> <p>The events reported in this data set have also undergone a full tri-dimensional reconstruction of the neutrino vertex in the nuclear emulsion films. A total of 5603 fully reconstructed neutrino interactions were reconstructed in the 0mu and 1mu categories. A search for possible decay topologies was applied to this sample of 5603 events with a procedure explained in the description of the <a href=\"/record/3900\">emulsion data</a>. A final selection was performed by applying kinematical cuts which selected the ten events reported in <a href=\"/record/3900\">this sample</a>.</p>  "
```

Comment:
We have to check what makes sense to be added as a link here in the last paragraph. Ditto for record 10000 below.

Reply:
The two links in the record 10000 (ED tau) should point to the record 10001 (ECC tau), not to the record 3900 (ED multiplicity).